packages/types/src/documents/account/account.ts

```diff
@@ -46,6 +47,7 @@ export interface Account extends CreateAccount {
   tier: string // deprecated
   planType?: PlanType
   planTier?: number
+  planDuration?: PriceDuration
```

Comment:
Mentioned on the other PR. I think we should just rename this type `Duration` now